libqtile/widget/tasklist.py

```diff
@@ -327,6 +348,14 @@ def update(self, window=None):
         if not window or window in self.windows:
             self.bar.draw()
 
+    def skip_taskbar(self, windows):
+        """
+        Check every window's _NET_WM_STATE property for _NET_WM_STATE_SKIP_TASKBAR
+        and remove that window from the list to be shown on the taskbar.
+        This is mostly for GTK programs. Qt doesn't seem to support this
+        """
+        return {w for w in windows if "_NET_WM_STATE_MAXIMIZED_VERT" not in w.window.get_net_wm_state()}
```

Comment:
I thought we were filtering out windows that had `_NET_WM_STATE_SKIP_TASKBAR`. Why are you checking for `_NET_WM_STATE_MAXIMIZED_VERT`?

Reply:
lol my bad. I dont have any programs that use skip task bar so I used the other one for testing and forgot.

Reply:
No problem.

Would be good if you could check it with a program that does set that property. I think, in the google group chain, someone mentioned a terminal that lets you set it.

Reply:
confirmed working by running 
`xprop -id "$windowid" -f _NET_WM_STATE 32a -set _NET_WM_STATE _NET_WM_STATE_SKIP_TASKBAR `